skbio/stats/composition.py

```diff
@@ -673,7 +779,7 @@ def centralize(mat):
     return perturb_inv(mat, cen)
 
 
-@experimental(as_of="0.4.1")
+@experimental(as_of="0.5.1-dev")
```

Comment:
please revert -- this is saying that `ancom` has an experimental API that was added in version 0.4.1. For more info about API stability annotations, see http://scikit-bio.org/docs/latest/user/api_stability.html